bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_cuts.f

```diff
@@ -0,0 +1,1034 @@
+c state particles
+      integer istatus(nexternal)
+c This is an array with (simplified) PDG codes for the particles. Note
+c that channels that are combined (i.e. they have the same matrix
+c elements) are given only 1 set of PDG codes. This means, e.g., that
+c when using a 5-flavour scheme calculation (massless b quark), no
+c b-tagging can be applied.
+      integer iPDG(nexternal)
+c The array of the momenta and masses of the initial and final state
+c particles in the lab frame. The format is "E, px, py, pz, mass", while
+c the second dimension loops over the particles in the process. Note
+c that these are the (n+1)-body particles; for the n-body there is one
+c momenta equal to all zero's (this is not necessarily the last particle
+c in the list). If one uses IR-safe obserables only, there should be no
+c difficulty in using this.
+      double precision p(0:4,nexternal)
+c
+C     external functions that can be used. Some are defined in this
+C     file, others are in ./Source/kin_functions.f
+      REAL*8 R2_04,invm2_04,invm3_04,pt_04,eta_04,pt,eta
+      external R2_04,invm2_04,invm3_04,pt_04,eta_04,pt,eta
+c local integers
+      integer i,j
+c jet cluster algorithm
+      integer nQCD,NJET,JET(nexternal)
+      double precision pQCD(0:3,nexternal),PJET(0:3,nexternal)
+      double precision rfj,sycut,palg,amcatnlo_fastjetdmerge
+      integer njet_eta
+      integer mm
+c Photon isolation
+      integer nph,nem,k,nin
+      double precision ptg,chi_gamma_iso,iso_getdrv40
+      double precision Etsum(0:nexternal)
+      real drlist(nexternal)
+      double precision pgamma(0:3,nexternal),pem(0:3,nexternal)
+      logical alliso
+c Sort array of results: ismode>0 for real, isway=0 for ascending order
+      integer ismode,isway,izero,isorted(nexternal)
+      parameter (ismode=1)
+      parameter (isway=0)
+      parameter (izero=0)
+c The UNLOPS cut
+      double precision p_unlops(0:3,nexternal)
+      include "run.inc" ! includes the ickkw parameter
+      logical passUNLOPScuts
+c logicals that define if particles are leptons, jets or photons. These
+c are filled from the PDG codes (iPDG array) in this function.
+      logical is_a_lp(nexternal),is_a_lm(nexternal),is_a_j(nexternal)
+     $     ,is_a_ph(nexternal)
+
+      passcuts_user=.true. ! event is okay; otherwise it is changed
+
+C***************************************************************
+C***************************************************************
+C Cuts from the run_card.dat
+C***************************************************************
+C***************************************************************
+c
+
+      do i=0,nexternal
+         do j=0,nexternal
+            do k=0,nexternal
+               if ((abs(ipdg(i)).eq.11.or.abs(ipdg(i)).eq.13.or.
+     &              abs(ipdg(i)).eq.15).and.
+     &              (abs(ipdg(i))+1.eq.abs(ipdg(j))).and.
+     &              abs(ipdg(k)).eq.24) then
```

Comment:
Is this "&" correct?

Reply:
I am not really familiar with fortran. This & is needed to link the above and below lines in the IF?

Reply:
& is used to continue the previous line on the new one, so it does not really do anything.

I just took the old approved configuration as reference: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_cuts.f#L81-L86

Reply:
Yeah sorry I realised that after looking at other lines